dev-python/nbval/nbval-0.9.6.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7..9} )
+inherit distutils-r1
+
+DESCRIPTION="A py.test plugin to validate Jupyter notebooks"
+HOMEPAGE="https://github.com/computationalmodelling/nbval"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="
+	dev-python/pytest[${PYTHON_USEDEP}]
+	dev-python/six[${PYTHON_USEDEP}]
+	dev-python/jupyter_client[${PYTHON_USEDEP}]
+	dev-python/nbformat[${PYTHON_USEDEP}]
+	dev-python/ipykernel[${PYTHON_USEDEP}]
+	dev-python/coverage[${PYTHON_USEDEP}]"
```

Comment:
Do we definitely need coverage at runtime, actually?

Reply:
https://github.com/computationalmodelling/nbval/blob/master/setup.py#L28

Reply:
Fine 😞